### ISSUE TITLE
Using Zend-view with PHP STAN

### ISSUE THREAD

**Issue by vitorloureiro** [zendframework/zend-view#183]
Currently i am using the zend view in my project and i notice a very small problem with the docblock of the \Zend\View\Model\ViewModel class constructor.
Where it says @param  null|array|Traversable $variables should be @param  null|array|Traversable|ArrayAccess $variables

In the same constructor if no $variables is provider the constructor initializes it with one instance of arrayAccess.

**Comment by dennybrandes** [comment]
Yes, you are right, cause method setVariables, which is called, also define ArrayAccess in DocBlock

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas/laminas-view; a new issue has been opened at https://github.com/laminas/laminas-view/issues/5.
